### PR TITLE
Fixes RCDs working on transit space

### PR DESCRIPTION
**Pull request by Cyberboss**

### CODE REVIEW

File: code/game/turfs/space/transit.dm [hunk 1]
```diff
@@ -91,3 +91,6 @@
 
 	icon_state = "speedspace_ns_[state]"
 	transform = turn(matrix(), angle)
+
+/turf/open/space/rcd_vals(mob/user, obj/item/weapon/rcd/the_rcd)
```
Comment (coiax) [inline code/game/turfs/space/transit.dm]: This looks like you're applying it to all space turfs, rather than transit space turfs.

Comment (RemieRichards) [inline code/game/turfs/space/transit.dm]: he is, he is dumb

Comment (Cyberboss) [inline code/game/turfs/space/transit.dm]: Yeah 1 sec

**Comment by Tacolizard** [comment]
this is a feature removal

**Comment by octareenroon91** [comment]
Why does this make sense?

Did anyone come up with [s]ecret technique[s]?

**Comment by octareenroon91** [comment]
Now I see the bug report.

Is this going to make in-transit escape shuttle incidents more inconvenient?

**Comment by coiax** [comment]
@Cyberboss I think there's an existing check allowing construction on transit space tiles if they were part of the shuttle's original area. Could you borrow that so we could still repair shit?

**Pull request merged**